corehq/apps/tour/models.py

```diff
@@ -0,0 +1,34 @@
+import datetime
+from django.contrib.auth.models import User
+from django.db import models
+import json_field
+
+
+class GuidedTours(models.Model):
+    user = models.ForeignKey(
+        User,
+        on_delete=models.PROTECT,
+        unique=True
+    )
+    seen_tours = json_field.JSONField(
+        default={},
+    )
+
+    @classmethod
+    def has_seen_tour(cls, user, tour_slug):
+        guided_tour, _ = GuidedTours.objects.get_or_create(user=user)
+        return tour_slug not in guided_tour.seen_tours
```

Comment:
No, I do not think we should be concerned with the space cost. I think the benefits of this new table structure in terms of runtime performance as well as the ability to query the data with SQL will far outweigh the space tradeoff (if any).

Also, I'm guessing that by the time space becomes an issue we'll be able to delete some records for old tours that are no longer in use if necessary.